### PR TITLE
[prometheus-node-exporter] add containerSecurityContext

### PR DESCRIPTION
**Pull request by cgroschupp**
#### What this PR does / why we need it:
Add parameter containerSecurityContext to add a securityContext to a container. To allow e.g. to set the capabilities "SYS_TIME" for the node_timex_sync_status metric.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by gkarthiks — APPROVED** [review]

**Comment by cgroschupp** [comment]
@gianrubio @vsliouniaev friendly ping for review.

**Review by monotek — DISMISSED** [review]

**Review by vsliouniaev — APPROVED** [review]
I thin you'll need to bump the chart version

File: charts/prometheus-node-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.0.1
 description: A Helm chart for prometheus node-exporter
 name: prometheus-node-exporter
-version: 1.12.0
+version: 1.13.0
```
Comment (gkarthiks) [inline charts/prometheus-node-exporter/Chart.yaml:5]: Hi @cgroschupp there's already a `1.13.0` [refer here](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/Chart.yaml#L5). Please bump the chart version.

Comment (cgroschupp) [reply charts/prometheus-node-exporter/Chart.yaml]: done

**Review by gkarthiks — DISMISSED** [review]

**Review by bismarck — APPROVED** [review]
lgtm

**Pull request merged**